torch/nn/quantized/modules/linear.py

```diff
@@ -142,8 +157,8 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             self.bias.copy_(state_dict[prefix + 'bias'])
             state_dict.pop(prefix + 'bias')
         state_dict.pop(prefix + 'weight')
-        super()._load_from_state_dict(state_dict, prefix, local_metadata, False,
-                                      missing_keys, unexpected_keys, error_msgs)
+        super(Linear, self)._load_from_state_dict(state_dict, prefix, local_metadata, False,
```

Comment:
Do we need special handling for bias? Can we delete the lines
```python
   if prefix + 'bias' in state_dict:
            self.bias.copy_(state_dict[prefix + 'bias'])
            state_dict.pop(prefix + 'bias')
```

Reply:
I am not sure why this was added. @jerryzh168 any idea?

Reply:
Actually, looks like the if condition exists because bias is optional.

Reply:
yeah bias is optional